devops/roles/analytics-druid/vars/pg_pass.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
The postgres server password should go into the ansible secrets file for each environment. This file will not be required.